Dockerfile.ci

```
@@ -104,6 +104,7 @@ RUN mkdir -pv /usr/share/man/man1 \
 
 COPY scripts/docker/*.sh /scripts/docker/
 RUN bash /scripts/docker/install_mysql.sh dev \
+    && /scripts/docker/install_mssql.sh \
```

```suggestion
    && bash /scripts/docker/install_mssql.sh \
```
For consistency. See previous line.

---

Right!